health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/timelinePopUpWrapper.js

```
@@ -0,0 +1,780 @@
+                },
+                "rating": 0,
+                "escalated": false
+            }
+        ],
+        "totalCount": 0
+    }
+    // Manage timeline data
+    const [timelineSteps, setTimelineSteps] = useState([]);
+
+    useEffect(() => {
+        if (hardCodeData && hardCodeData.ProcessInstances) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)